src/public/js/api.js

```diff
@@ -238,6 +238,8 @@ const onGetReceipt = async (bankTransactionId) => {
   }
 };
 
+const formatAsCurrency = (amount, currency = 'GBP')  => new Intl.NumberFormat(undefined, { style: 'currency', currency: currency }).format(amount);
```

Comment:
Does specifying undefined here mean something special (like take local locality)? My assumption would be en-GB or whatever it is 🤷

Reply:
> undefined (or omitted): The implementation's default locale will be used.

I mostly put undefined as it's optional and I thought it's better than hardcoding one.

Reply:
fair enough :)